.eslintrc.json

```diff
@@ -0,0 +1,43 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": ["react-app", "prettier"],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "prettier"
+    ],
+    "rules": {
+        "prettier/prettier": [
+            "error", {
+                "printWidth": 80,
+                "trailingComma": "es5",
+                "semi": true,
+                "jsxSingleQuote": true,
+                "singleQuote": true,
+                "useTabs": true,
```

Comment:
Please, remember one golden rule: **never use tabs for indentations**!
The reason is quite simple: every system/browser has its own way for encoding and it may interpreter tab in different way. It will make some discrepancies in code reading. Let's remove this rule, please.

Example how this rule looks like when you remove tabs and have `tabWidth` set to `2`:
<img width="486" alt="Снимок экрана 2023-02-12 в 12 08 26" src="https://user-images.githubusercontent.com/43034008/218302265-4f60adcf-612a-40cd-8495-0b8ef8585fdd.png">